packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -344,6 +345,27 @@ export interface Behavior {
    */
   readonly maxTtlSeconds?: number;
 
+  /**
+   * Declares associated lambda@edge functions for this distribution behaviour.
+   *
+   * @default undefined
+   */
+  readonly lambdaFunctionAssociations?: LambdaFunctionAssociation[];
+
+}
+
+export interface LambdaFunctionAssociation {
+
+  readonly eventType?: LambdaEdgeEventType;
```

Comment:
Docstrings on all fields and the interface itself please.

What does it mean if both fields are omitted?

Reply:
I'll add docstrings on every field.
I've allowed to omit both fields, since its like that in the cloudfoundation template. To be honest, I did not tried what happened. Probably the deployment would fail and rollback.
Should the type requirements be the most sane or the technical base from cloudfoundation?

Reply:
Sane please :)